### PR TITLE
Add alt text to images

### LINKED ISSUE #768: Add alt text to images
It may be a hassle, but it would be nice if you could add alt text to the in-line images. This would make the book more accessible to people who use screen readers. An example of such an image that could benefit is the Fork button in this section: https://r-pkgs.org/git.html#pr-make. You're probably well aware how to do this, but just in case: https://www.rstudio.com/blog/knitr-fig-alt/.

### PR DESCRIPTION
**Pull request by ateucher**
This adds alt text to remaining images throughout the book, and updates two of the images in the workflow chapter.

Closes #768

### CODE REVIEW

**Comment by ateucher** [comment]
@batpigandme - I am starting to add alt text to images and wondered if you might be able to have a look at this as a starting point?

File: structure.Rmd [hunk 1]
```diff
@@ -91,6 +91,11 @@ This diagram summarises the files present in the top-level directory for source,
 #| out-width: ~
 #| fig-cap: >
 #|   Side-by-side comparison of source, bundled, and binary package.
+#| fig-alt: >
+#|   Side-by-side comparison of source, bundled, and binary package. 
+#|   The flow of files (location and format) from package source to 
+#|   bundled to binary state is shown. This is described in
+#|   detail in the appropriately named sections of this chapter
```
Comment (batpigandme) [inline structure.Rmd:98]:
```suggestion
#|   detail in the appropriately named sections of this chapter.
```

**Review by batpigandme — APPROVED** [review]
These look great, Andy. The only change I made was to add a period at the end of one of them, since that creates a useful pause for screenreaders.

**Review by jennybc — APPROVED** [review]
Fantastic! Thanks very much.

**Pull request merged**